src/components/Dialog/ActionDialogContentInner.tsx

```diff
@@ -70,33 +79,37 @@ export const ActionDialogContentInner: VFC<ActionDialogContentInnerProps> = ({
   if (actionTheme === 'secondary') ActionButton = SecondaryButton
   if (actionTheme === 'danger') ActionButton = DangerButton
 
+  const requestProcessing = responseMessage && responseMessage.status === 'processing'
+
   return (
     <>
       <Title themes={theme} ref={titleRef}>
         {title}
       </Title>
       <Body offsetHeight={offsetHeight}>{children}</Body>
-      <Bottom themes={theme} ref={bottomRef}>
-        <SecondaryButton onClick={onClickClose} disabled={closeDisabled}>
-          {closeText}
-        </SecondaryButton>
-        <ActionButton onClick={handleClickAction} disabled={actionDisabled}>
-          {actionText}
-        </ActionButton>
-      </Bottom>
+      <ActionArea themes={theme} ref={bottomRef}>
+        <ButtonArea themes={theme}>
+          <SecondaryButton onClick={onClickClose} disabled={closeDisabled || requestProcessing}>
+            {closeText}
+          </SecondaryButton>
+          <ActionButton onClick={handleClickAction} disabled={actionDisabled || requestProcessing}>
+            {actionText}
+          </ActionButton>
+        </ButtonArea>
+        <ResponseMessage themes={theme} responseMessage={responseMessage} />
```

Comment:
`<ResponseMessage>` are important change that are not given  keyboard focus, so we'd better wrap it with live region.

`<ResponseMessage>` はキーボードフォーカスが与えられない重要な変更なので、ライブリージョンで囲んだ方が良いでしょう。

```suggestion
        <div role="alert">
          <ResponseMessage themes={theme} responseMessage={responseMessage} />
        </div>
```

Reply:
修正した